ocs_ci/ocs/jenkins.py

```diff
@@ -154,6 +154,22 @@ def wait_for_build_to_complete(self, timeout=900):
                         self.print_completed_builds_results()
                         raise UnexpectedBehaviour(error_msg)
 
+    def get_builds_sorted_by_number(self, project):
+        """
+        Get builds per project and sort builds by build name number
+        Args:
+            project (str): project name
+
+        return:
+            jenkins_builds_sorted (lst): list of build (OCS obj)
+        """
```

Comment:
Please fix doc strings according to the guidelines. Check out the example in our coding guidelines.